frontend/playwright-tests/diabetes.nightly.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test } from '@playwright/test'
+
+test('test', async ({ page }) => {
```

Comment:
I left a tagged comment in the last PR; for these can you be sure to add a descriptive string for the test? It helps when running the full suite to see which ones are passing/failing

Reply:
Got it! I'll go back and look for any others I may have missed and fix them too